README.md

```diff
@@ -58,6 +58,7 @@ $ kubectl get opsrc upstream-community-operators -o=custom-columns=NAME:.metadat
 NAME                           PACKAGES
 upstream-community-operators   federationv2,svcat,metering,etcd,prometheus,automationbroker,templateservicebroker,cluster-logging,jaeger,descheduler
 ```
+**_Note_**: Please do not install [upstream-community-operators](deploy/examples/upstream.operatorsource.cr.yaml) and [community-operators](deploy/examples/community.operatorsource.cr.yaml) `OperatorSources` on the same cluster. The rule of thumb is to install [community-operators](deploy/examples/community.operatorsource.cr.yaml) on OpenShift clusters and [upstream-community-operators](deploy/examples/upstream.operatorsource.cr.yaml) on plain vanilla Kubernetes clusters.
```

Comment:
Perhaps, in addition to this, the upstream example cr should move to a new folder `deploy/upstream/examples/`? The `deploy/upstream/` folder already exists.

Reply:
Addressed